src/Apps/Settings/Routes/SavedSearchAlerts/useDeleteAlert.ts

```diff
@@ -0,0 +1,21 @@
+import { useMutation } from "Utils/Hooks/useMutation"
+import { graphql } from "react-relay"
+import { useDeleteAlertMutation } from "__generated__/useDeleteAlertMutation.graphql"
+
+export const useDeleteAlert = () => {
+  return useMutation<useDeleteAlertMutation>({
+    mutation: graphql`
+      mutation useDeleteAlertMutation($input: deleteAlertInput!) {
+        deleteAlert(input: $input) {
+          responseOrError {
+            ... on DeleteAlertSuccess {
```

Comment:
You can see here (and the other mutations) how we're not really handling errors at all (possibly a circumstance of handling errors in the stitched schema being odd - a win of the migration will be improved error handling!). If we were, I'd expect to see a `... on DeleteAlertError` for informative feedback to the user (altho, I suppose the _lack_ of a payload for the 'success' fragment could be interpreted in that way as well for a generic 'Something Went Wrong' toast).